### PR TITLE
Fix the setuptools_scm issue

### PR DESCRIPTION
**Pull request by pcmoritz**
#1782

### CODE REVIEW

**Comment by robertnishihara** [comment]
The current fix doesn't seem to work.

File: python/build-wheel-macos.sh [hunk 1]
```diff
@@ -48,9 +48,11 @@ for ((i=0; i<${#PY_VERSIONS[@]}; ++i)); do
   PIP_CMD="$(dirname $PYTHON_EXE)/pip$PY_MM"
 
   pushd python
+    # Upgrade pip.
+    PATH=$MACPYTHON_PY_PREFIX/$PY_MM/bin:$PATH $PYTHON_EXE -m pip install --upgrade pip setuptools wheel
```
Comment (robertnishihara) [inline python/build-wheel-macos.sh]: you shouldn't change the PATH, just use `$PIP_CMD`

Comment (robertnishihara) [inline python/build-wheel-macos.sh]: Also, are you sure it's necessary to upgrade all of those?

Comment (pcmoritz) [reply python/build-wheel-macos.sh]: I'll try just pip

**Review by robertnishihara — APPROVED** [review]

**Pull request merged**